misc/plot_rmm.py

```diff
@@ -0,0 +1,77 @@
+# Usage: python3 plot_rmm.py rmm_bench.csv --save-dir=plots --logx --smooth 3
+import argparse
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+
+OPS_ORDER = [
+    "rank1",
+    "rank0",
+    "select1",
+    "select0",
+    "rank10",
+    "select10",
+    "excess",
+    "fwdsearch",
+    "bwdsearch",
+    "rmq_pos",
+    "rmq_val",
+    "rMq_pos",
+    "rMq_val",
+    "mincount",
+    "minselect",
+]
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("csv")
+    ap.add_argument("--save-dir", default="plots")
```

Comment:
If this is intended to be used by others, please add descriptions to the arguments. Also a a description for the whole script is also good. AI's are quite good for this kind of tasks